src/Template/Loggers/IOHprofiler_csv_logger.cpp

```diff
@@ -125,13 +125,21 @@ void IOHprofiler_csv_logger::write_header() {
 /// To update info of current working problem, and to write headline in corresponding files.
 void IOHprofiler_csv_logger::target_problem(int problem_id, int dimension, int instance, std::string problem_name){
   /// Handle info of the previous problem.
-  if(infoFile.is_open()) write_info(this->instance,this->found_optimal[0],this->optimal_evaluations);
+  if(infoFile.is_open()) write_info(this->instance,this->best_y[0],this->best_trasnformed_y[0],this->optimal_evaluations,
```

Comment:
please make sure to always have a space after a comma to keep long lines readable